.github/workflows/js-ci.yml

```diff
@@ -1,11 +1,4 @@
-on:
-  push:
-    branches:
-      - master
-    tags:
-      - "*"
-  pull_request:
```

Comment:
I'm not sure it's really needed, but https://github.com/quiltdata/quilt/pull/3001